### DISCUSSION TITLE
[Suggestion] display YouTube Thumbnail when YouTube link is detected?

### DISCUSSION THREAD

**Discussion by desirable-difficulty** [usememos/memos#2473] [Ideas]
currently when I paste a YouTube link, it just shows the link itself.
I'd suggest the twitter/x way, when YouTube link is detected, show the thumbnail automatically.

**Comment by MehadND** [comment]
Hi @desirable-difficulty, can you please check this [PR](https://github.com/usememos/memos/pull/2812) and let me know if this the kind off thing that you were looking for.
